examples/amiibo/frontend/workflows/amiibo/src/index.tsx

```diff
@@ -15,9 +15,9 @@ const register = (): WorkflowConfiguration => {
     displayName: "Amiibo",
     routes: {
       landing: {
-        path: "/",
+        path: "/lookup",
```

Comment:
updated b/c docs say to navigate to http://localhost:3000/amiibo/lookup

https://clutch.sh/docs/development/feature/#test-1